### ISSUE TITLE
Feature Request: Absolute Attendance in Number, not Percent

### ISSUE THREAD

**Issue by RaefWolfe** [sawprogramming/WoW-Raid-Manager#49]
At a glance attendance percentage is useful for my guild to track who is maintaining attendance requirements after they're already received a promotion. But we require a certain number of raids attended before promoting a member in the first place. It'd be nice to have an additional dropdown / settings option to show total attendance as a raw number (if they have 5 entries, and only 1 entry is an absent, it would show the number 4 at a glance).

**Comment by sawprogramming** [comment]
I'll try take a look this weekend and see the effort it would take.

**Comment by sawprogramming** [comment]
Hey, sorry it took awhile! I finally sat down and added this feature in. Let me know if you like it!

**Comment by RaefWolfe** [comment]
I'm having some issues, but this looks like just what I was hoping for.

 - In spite of everyone on the attendance sheet having only positive attendance (no absences currently), the player breakdown shows "absent" rather than "present" attendance in the new table.

- Changing to "absolute" attendance in the dropdown doesn't appear to have any effect (still only shows 100%, rather than say 4 or 5)

I disabled the plugin, installed the updated package on top of the old plugin, and then enabled the plugin. Did I miss a step anywhere? (I did download a backup of my data before updating just in case I needed it.)

https://www.ministryofdefense.net/raiding/raid-attendance/
